pkg/debug/scripts/collect-info.sh

```diff
@@ -234,6 +234,17 @@ collect_network_info()
     echo "- done network info"
 }
 
+collect_pillar_memory_backtraces()
+{
+    echo "- pillar memory backtraces"
+
+    eve http-debug > /dev/null 2>&1
+    eve exec debug curl --retry-all-errors --retry 5 -m 5 "http://127.1:6543/debug/pprof/heap?debug=1" > "$DIR/pillar-memory-backtraces"
```

Comment:
Please redirect the stderr to the file/null as well, otherwise we see the curl output with download progress.

Reply:
Done